main.tf

```diff
@@ -0,0 +1,77 @@
+module "label" {
+  source    = "git::https://github.com/cloudposse/tf_label.git?ref=tags/0.2.0"
+  namespace = "${var.namespace}"
+  name      = "${var.name}"
+  stage     = "${var.stage}"
+}
+
+resource "aws_db_instance" "default" {
+  identifier                  = "${module.label.id}"
+  name                        = "${var.database_name}"
+  username                    = "${var.database_user}"
+  password                    = "${var.database_password}"
+  port                        = "${var.database_port}"
+  engine                      = "${var.rds_engine_type}"
+  engine_version              = "${var.rds_engine_version}"
+  instance_class              = "${var.rds_instance_class}"
+  allocated_storage           = "${var.rds_allocated_storage}"
+  vpc_security_group_ids      = ["${aws_security_group.default.id}"]
+  db_subnet_group_name        = "${aws_db_subnet_group.default.name}"
+  parameter_group_name        = "${aws_db_parameter_group.default.name}"
+  multi_az                    = "${var.rds_is_multi_az}"
```

Comment:
Why introduce a new `rds_is_` prefix? It's not used consistently. Prefer to mimic the original variable names where we simply pass them through to the underlying resource. If we do a data transformation on the variable, then we might want to call it something different.